openmdao/solvers/nonlinear/newton.py

```diff
@@ -130,14 +130,9 @@ def _set_solver_print(self, level=2, type_='all'):
         if self.linesearch is not None:
             self.linesearch._set_solver_print(level=level, type_=type_)
 
-    def _iter_get_norm(self):
+    def _run_apply(self):
         """
-        Return the norm of the residual.
-
-        Returns
-        -------
-        float
-            norm.
+        Run the the apply_nonlinear method on the system.
         """
         recording_iteration_stack.append(('_iter_get_norm', 0))
```

Comment:
I thought all the instances of the `'_iter_get_norm'` recording tag were being replaced by the `'_run_apply'`?